src/Security/SecretStoreBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Arcus.Security.Providers.AzureKeyVault;
+using Arcus.Security.Providers.AzureKeyVault.Authentication;
+using Arcus.Security.Providers.AzureKeyVault.Configuration;
+using Microsoft.Extensions.Hosting;
+
+namespace Arcus.Security.Startup.Security 
+{
+    public static class SecretStoreBuilderExtensions
+    {
+        public static SecretStoreBuilder AddAzureKeyVaultProvider(this SecretStoreBuilder builder, IKeyVaultAuthentication authentication, IKeyVaultConfiguration configuration)
+        {
+            return builder.AddProvider(new KeyVaultSecretProvider(authentication, configuration));
+        }
+
+        public static SecretStoreBuilder AddEnvironmentVariableProvider(this SecretStoreBuilder builder)
+        {
+            System.Environment.GetEnvironmentVariables();
```

Comment:
This is odd, why doesn't it add the provider?

Reply:
😄 My bad, was expirimenting, but changed the approach. I'll have to remove this line.

Reply:
Hehe ok!